product_docs/docs/epas/15/epas_compat_sql/65a_merge.mdx

```diff
@@ -25,7 +27,7 @@ WHEN NOT MATCHED THEN
 
 `MERGE` allows you to select rows from one or more sources for update or insertion into a table. You specify the join condition to determine whether to update or insert into the target table. You specify conditional UPDATE and INSERT statements using the WHERE clause in the MERGE statement.
 
-It allows you to combine multiple operations and avoid multiple INSERT, UPDATE, and DELETE DML statements.
+`MERGE` provides a single SQL statement that can conditionally INSERT, UPDATE or DELETE rows, a task requiring multiple procedural language statements.
```

Comment:
I don't follow this sentence. I think you are trying to say that the Merge statement allows you to do something in a single statement that otherwise would take multiple statements, but this isn't how it reads. 

Here is a suggested rewrite: 
```suggestion
`MERGE` provides a single SQL statement that can conditionally INSERT, UPDATE, or DELETE rows, a task which would otherwise require multiple procedural language statements.
```